frameworks/Clojure/reitit/docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '2'
```

Comment:
Hi,
you don't need to embed PostgreSQL in the framework's `Docker` file. The TFB toolset is taking care of starting the appropriate database container for the tests. The configurations are [accessible here](https://github.com/TechEmpower/FrameworkBenchmarks/tree/master/toolset/databases).

Reply:
Thanks! Removed that part and polished the implementation.